workshop/local-participant-information.md

```diff
@@ -2,65 +2,48 @@
 <!-- Please update this file to reflect logistics for the workshop being taught. -->
 
 
-
 ## Workshop Location
 
-**The workshop will be held at** [**One Bala Plaza, Bala Cynwyd, PA 19004**](https://www.google.com/maps/place/One+Bala+Plaza/@40.0073079,-75.2221085,17z/data=!3m1!4b1!4m5!3m4!1s0x89c6b89e7102a3b5:0x77dce0b150a7df52!8m2!3d40.0073798!4d-75.2199724?utm_campaign=CCDL_Workshops&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz--pRa135WXpaamTCddydLZAXVv6QwBlCpR9HqVWrGN1EmHMVqdhOeW6wlCMhaR6eCaT1Ekh), just outside of Philadelphia.
-We will meet in the Amenity Suite located on the lower level.
-Enter the building through the Center Lobby and take the elevator down one floor (labeled "LL").
-The Amenity Suite is the first room you will see after exiting the elevator.
-Please note that Bala Plaza is an office park with multiple buildings.
-If you have trouble finding the building or the room on the first day of training, please use Slack to [direct message](/software-setup/slack-procedures.html#using-direct-messages-during-training) `Jen O'Malley`.
- {% comment %}
- NOTE: `html` relative link above is used since this is an included file.
- {% endcomment %}
-
-## Travel logistics
-### Transportation
+The Bulk RNA-Sequencing and Reproducible Research Practices Workshop will be held at the University of Minnesota in the building located at **717 Delaware Street SE, Minneapolis, MN 55414. 
+We will meet in Room 105.**
+Enter through the front main entrance.
 
-**Driving**
+If you have trouble finding the building or the room on the first day of training, please use Slack to direct message `Jen O'Malley`.
 
-* There is free onsite parking available both in front of and behind One Bala Plaza.
+## Pre-workshop Information
 
-**Public transportation**
+**Please review and sign our waivers:**
 
-* SEPTA is the public transportation system in Philadelphia and the surrounding area.
-* At the airport, and at some SEPTA train stations, you can purchase a SEPTA key card. There are a variety of fare options that can be loaded onto a key card for use on public transportation. SEPTA also accepts cash for travel on any of its transit services.
-* Please refer to the [SEPTA website](https://www5.septa.org/travel/) for more information. (**Note:** The [SEPTA trip planner](https://beta-plan.septa.org/#/) may be a helpful tool if you plan to travel around the city during your stay.)
+* [Training waiver](https://app.hellosign.com/s/2o2yghCj)
+* [Data use waiver](https://app.hellosign.com/s/7AZ7TWXe)
```

Comment:
I actually don't think we need this - no ScPCA data is used for a bulk RNA-seq workshop!